modules/ding_frontend/ding_frontend.pages_default.inc

```diff
@@ -811,47 +847,12 @@ search/node',
     $display->content['new-d9c1432b-77cc-432c-987a-eb64569a0b88'] = $pane;
     $display->panels['header'][2] = 'new-d9c1432b-77cc-432c-987a-eb64569a0b88';
     $pane = new stdClass();
-    $pane->pid = 'new-37d18c98-64ec-448f-ac62-ad0cc199fe06';
-    $pane->panel = 'header';
-    $pane->type = 'block';
-    $pane->subtype = 'ding_wayf-login';
```

Comment:
Note that these blocks have been readded in #328 so I do think they should be deleted.